microovn/api/certificates/regenerate_ca.go

```diff
@@ -29,7 +29,7 @@ func regenerateCaPut(s state.State, r *http.Request) response.Response {
 	responseData := types.NewRegenerateCaResponse()
 
 	// Check that this is the initial node that received the request and recreate new CA certificate
-	if client.IsNotification(r) {
+	if !client.IsNotification(r) {
```

Comment:
Wow, 5b42b2ed7a291de6c5a5551b5b3edfbb35a45adb even inverted this check when transitioning from `IsForwardedRequest` to `IsNotification`, which is not great.